### PR TITLE
Admin's "Create Post" and "Get Feeds" UI is created.

### PR DESCRIPTION
**Pull request by haja2611**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,5 @@
-REACT_APP_API_URL=https://api.melbeez.com
+REACT_APP_API_URL=https://api.com
```
Comment (worldynllc) [inline .env]: Is there any reason to change the base URL? Can you revert it?

**Review by worldynllc — COMMENTED** [review]
Please change the base URL for REACT_APP_API_URL to the original. Later, we can change the config based on the environment.

https://create-react-app.dev/docs/adding-custom-environment-variables/

File: .env (hunk 1 above)
Comment (haja2611) [reply .env]: Is changed only for the security purpose.Because i will share public repository. I can revert it.

Comment (worldynllc) [reply .env]: Please revert to the original.

**Review by worldynllc — APPROVED** [review]

**Pull request merged**